docs/core/tutorials/using-on-macos.md

```diff
@@ -49,95 +44,79 @@ terminal window, if you prefer).
 
 By the time we're done, you'll create three projects: a library project,
 tests for that library project, and a console application that makes
-use of the library. You'll follow a standard folder structure for
-the three projects. Following this standard folder structure
-means that the .NET Core SDK tools understand the relationship between
-your production code projects and your test code projects. That makes
-your development experience more productive.
+use of the library. 
 
 Let's start by creating those folders. In the terminal, create a 'golden'
-directory. Under that directory create `src` and `test`
-directories. Under `src` create `app` and `library` directories. In `test`
-create a `test-library` directory. You can do this either using the terminal
-in VS code, or by clicking on the parent folder in VS Code and selecting the
-"New Folder" icon.
+directory. In VS Code, open the *golden* directory. This directory is the root of your solution. Run the `dotnet new` command to create a new solution:
 
-In VS Code, open the 'golden' directory. This directory is the root of your solution.
+```
+dotnet new sln
+```
 
-Next, create a `global.json` file in the root directory for your solution.
-The contents of `global.json` are:
+This command creates a *golden.sln* file for the entire solution.
 
-```json
-{
-    "projects": [
-        "src",
-        "test"
-    ]
-}
-```
+Your next task is to create the library. In the terminal window
+(either the embedded terminal in VS code, or another terminal),
+cd to *golden/* and type the command:
 
-At this point, your directory tree should look like this:
+```
+dotnet new classlib library
+```
 
+This creates a library project, with two files: *library.csproj* and
+*Class1.cs* in the *library* directory. You want that library project
+included in your solution. Run the `dotnet sln` command to add the newly
```

Comment:
Fixed. Note: This now must wait until the preview 3 folder merges with the tools folder.